File: CS/MultipleCheckExample/ViewModel/TaskViewModel.cs
using MultipleCheckExample.Model;
using System;
using System.Collections.ObjectModel;
using System.Linq;

namespace MultipleCheckExample
{
    public class TaskViewModel
    {
        public ObservableCollection<Task> List { get; set; }
        public TaskViewModel() {
            List = new ObservableCollection<Task>();
            for (int i = 0; i < 5; i++) {
                List.Add(Task.NewTask(i));
            }
        }
    }
}

File: VB/MultipleCheckExample/ViewModel/TaskViewModel.vb
Imports MultipleCheckExample.Model
Imports System
Imports System.Collections.ObjectModel
Imports System.Linq

Namespace MultipleCheckExample
	Public Class TaskViewModel
		Public Property List() As ObservableCollection(Of Task)
		Public Sub New()
			List = New ObservableCollection(Of Task)()
			For i As Integer = 0 To 4
				List.Add(Task.NewTask(i))
			Next i
		End Sub
	End Class
End Namespace
